CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+- Updated PhaseBanner to include Feedback component
```

Comment:
Please move this under an appropriate header. Either `## Added` or `## Changed` should be fine